Add Cloth Cap exam using FRR

georgelzrc opened the pull request:
Added 2013-12-20 Cloth Cap exam using FRR

lorenzo93 commented on Exam Labs/2013-12-20 Cloth Cap/lab/as10r1/etc/frr/frr.conf:
```diff
@@ -0,0 +1,43 @@
+
+router bgp 10
+
+! no bgp ebgp-requires-policy
```
Please, remove commented comments.

lorenzo93 commented on Exam Labs/2013-12-20 Cloth Cap/lab/as10r1/etc/frr/frr.conf:
```diff
@@ -0,0 +1,43 @@
+router bgp 10
+
+! no bgp ebgp-requires-policy
+no bgp network import-check
```
Useless command because it announces the networks of the interfaces

lorenzo93 commented on Exam Labs/2013-12-20 Cloth Cap/lab/as10r1/etc/frr/frr.conf:
```diff
@@ -0,0 +1,43 @@
+neighbor 10.14.0.14 prefix-list mandaTutto in
+neighbor 10.14.0.14 route-map localPref in
+
+ip prefix-list rifiutaTutto deny any
```
Please, define the prefix-lists, access-lists and route-maps with english names to be more inclusive

lorenzo93 commented on Exam Labs/2013-12-20 Cloth Cap/lab/as10r1/etc/frr/frr.conf:
```diff
@@ -0,0 +1,43 @@
+ip prefix-list mandaTutto permit any
+
+route-map localPref permit 10
+match ip address myAccessList
```
Please, add indentation to improve readability. (you can find the correct indentation running the command show running-config inside vtysh)

lorenzo93 commented on Exam Labs/2013-12-20 Cloth Cap/lab/as10r1/etc/frr/frr.conf:
```diff
@@ -0,0 +1,43 @@
+neighbor 10.14.0.14 description Router as14r1
+
+
+network 10.20.0.0/24
```
Please, redistribute connected instead of announce networks in an explicit way

lorenzo93 commented on Exam Labs/2013-12-20 Cloth Cap/lab/as20r1.startup:
```diff
@@ -0,0 +1,6 @@
+ifconfig eth1 20.0.0.1/30 up
+ifconfig eth2 10.20.0.1/24 up
+ifconfig eth3 20.2.0.1/24 up
+ifconfig lo:1 1.1.1.1 netmask 255.255.255.255 up
```
Loopback peering is not needed :)

lorenzo93 commented on Exam Labs/2013-12-20 Cloth Cap/lab/lab.conf:
```diff
@@ -0,0 +1,33 @@
+as20r1[0]=L
```
Please, add at the start of the lab.conf the metadata (you can see an example other labs) and feel free to put your name as author :)

lorenzo93 reviewed (changes requested):
Hi @georgelzrc 

thanks very much for your work, we appreciate it a lot.
I've made some comments on your code. Notice that, some of them are just on one file but they should be seen as "global" comments.

georgelzrc commented:
Hi @lorenzo93 I made the changes you asked for.
These inlcude:
- Only use English names
- Remove commented and unnecessary lines of code
- Add metadata in lab.conf
- Do not use loopback interfaces for peering

My colleague Francesco and I are pleased with your appreciation for our hard work in making this lab.

lorenzo93 commented:
Hi @georgelzrc ,

I've checked your work.

Just one minor comment about the Cloth Cap exam.
The correct indentation for the `frr.conf` is a little bit different. (You can find the correct indentation running the command `show running-config` inside `vtysh`).

On the other hand, about the Alien exam:
1. If you are able to, remove the submission instructions from the pdf. If you want I can provide you the modified PDF version :)
2. Same comment about indentation as the Cloth Cap exam.

Remember also, when you'll apply this changes to re-pack the `.zip` files.

PS: for the next time, if you add two different labs, please open two different PRs :)

georgelzrc commented:
Hi @lorenzo93 I improved the indentation as you suggested and added some comments in the configurations to allow for better readability. I also re-packed .zip files and removed the submission instructions in alien :)
